dts/bindings/serial/nxp,kinetis-uart.yaml

```diff
@@ -16,3 +16,8 @@ properties:
 
   pinctrl-0:
     required: true
+
+  no-error-irq:
+    type: boolean
+    description: |
+      If the SoCs don't have a separate interrupt id for error IRQ.
```

Comment:
this is not a grammatically complete sentence.

Reply:
updated